api/src/main/resources/api/service_openapi.yaml

```diff
@@ -20,6 +20,57 @@ paths:
         500:
           description: Service is broken
 
+  '/underlays':
+    get:
+      # TODO authorization
+      security: []
+      summary: Lists the underlays.
+      operationId: listUnderlays
+      tags: [ Underlays ]
+      responses:
+        200:
+          $ref: '#/components/responses/ListUnderlaysResponse'
+
+  '/underlays/{underlay}':
+    parameters:
+      - $ref: '#/components/parameters/Underlay'
+    get:
+      # TODO authorization
+      security: []
+      summary: Get info on an underlay.
+      operationId: getUnderlay
+      tags: [ Underlays ]
+      responses:
+        200:
+          $ref: '#/components/responses/GetUnderlayResponse'
```

Comment:
Per https://cloud.google.com/apis/design/standard_methods#get  - I recommend returning the entity here rather than a wrapper. In my experience this has worked: if you find yourself having to include other things in the response object (aside from the resource), then probably you've deviated from a standard REST method.

Here and below for GetEntity

Reply:
I thought that's what I was doing. Don't I need the intervening "responses" object with `content: application/json:` around the schema? I could inline the `GetUnderlayResponse` object, I don't really know if there's a good reason to prefer or avoid that.

Reply:
Sorry, I misread this - I'm less accustomed to OpenAPI 3. As long as the response Entity is not wrapped - we should be good.